src/pages/starter-kit/checkout/payment-install.md

```diff
@@ -14,6 +14,12 @@ To begin using the payment module with the checkout starter kit, ensure that you
 
 For more ideas on how you can use the payment module, refer to [payment use cases](./payment-use-cases.md).
 
+## Prerequisites
+
+* Adobe Commerce on cloud infrastructure or on premises: 2.4.5+
```

Comment:
```suggestion
* Adobe Commerce on cloud infrastructure or on-premises: 2.4.5+
```

Reply:
The prerequisite block should be replaced with an includes. See https://github.com/AdobeDocs/commerce-extensibility/blob/main/src/pages/starter-kit/integration/receive-data.md?plain=1 for examples.

That way one source of truth can be referenced in each topic.

Reply:
This part is the only place required to declare the PaaS's supported Commerce version for the module installation. Do we still need to export them? Currently each module (payment, tax, shipping) has all different PHP + Commerce version combinations.